src/rpc/mining.cpp

```diff
@@ -768,6 +781,12 @@ static UniValue getblocktemplate(const JSONRPCRequest& request)
     result.pushKV("accumulatorhashes", mapaccumulatorhashes);
     result.pushKV("proofoffullnodehash", pblock->hashPoFN.GetHex());
 
+    if (pblock->IsProgPow()) {
+        result.pushKV("pprpcheader", pblock->GetProgPowHeaderHash().GetHex());
+        result.pushKV("pprpcepoch", progpow::get_epoch_number(pblock->nHeight));
```

Comment:
These need to be added to the getblocktemplate Result help. (other stuff needs to be added too, but not necessary for this PR; lots of cleanup is needed in the cli help sections)

Reply:
Added